dnx/src/Microsoft.DotNet.xunit.performance.runner.dnx/project.json

```diff
@@ -12,7 +12,7 @@
   "requireLicenseAcceptance": false,
 
   "dependencies": {
-    "Microsoft.DotNet.xunit.performance.run.core": "99.99.99-dev",
+    "Microsoft.DotNet.xunit.performance.run.core": "1.0.0-alpha-build0018",
```

Comment:
I don't think this is right. I pointed out something similar in a previous review to @ericeil.

These files are supposed to have the  "99.99.99-dev" version in them so that you can develop and test locally. The versions should get changed by the nightly build script. As part of the xunit.performance.msbuild file, there is a task which updates all version numbers. At the moment, it's quite simple, but it should be augmented to find all project.json files and replace "99.99.99-dev" with the desired package version. Take a look at what xunit itself does here.